root/static/scripts/common/components/EntityLink.js

```diff
@@ -292,20 +292,38 @@ $ReadOnlyArray<Expand2ReactOutput> | Expand2ReactOutput | null => {
     );
   }
 
-  if (showCaaPresence &&
-    entity.entityType === 'release' &&
-    entity.cover_art_presence === 'present') {
-    content = (
-      <>
-        <a href={'/release/' + entity.gid + '/cover-art'}>
+  if (showCaaPresence) {
+    console.log(entity);
+
+    if (entity.entityType === 'release' &&
+        entity.cover_art_presence === 'present') {
+      content = (
+        <>
+          <a href={'/release/' + entity.gid + '/cover-art'}>
+            <span
+              className="caa-icon"
+              title={l('This release has artwork in the Cover Art Archive')}
+            />
+          </a>
+          {content}
+        </>
+      );
+    }
+
+    if (entity.entityType === 'release_group' &&
+        entity.hasCoverArt /*:: === true */) {
```

Comment:
(following the previous change)
```suggestion
    if (entity.entityType === 'release_group' && entity.hasCoverArt) {
```